commcare/src/main/resources/webapp/messages/messages.properties

```diff
@@ -201,6 +201,7 @@ commcare.error=Error
 commcare.error.cases=Unable to retrieve the list of cases
 commcare.error.schema=Unable to retrieve application schema
 
+commcare.confirm.removeSetting=Are you sure you want to remove this setting?
```

Comment:
I think we should stick with names from UI, so I propose : "Are you sure you want to delete this configuration"

Moreover you can change the property name to "commcare.confirm.deleteConfig"